cockatrice/src/chatview/chatview.cpp

```diff
@@ -528,6 +559,14 @@ void ChatView::mouseMoveEvent(QMouseEvent *event)
 void ChatView::mousePressEvent(QMouseEvent *event)
 {
     switch (hoveredItemType) {
+        case HoveredGame: {
+            if ((event->button() == Qt::MiddleButton) || (event->button() == Qt::LeftButton)) {
+                // focus game in list if possible
```

Comment:
What is this supposed to do?

Reply:
nothing yet, I want your opinion on the changes to the gamesList first, after that I am planning to have this focus the game in the room's view as a way to provide the user with the stats of the game without introducing new ui elements